enterprise/frontend/src/embedding-sdk/components/public/MetabaseProvider.tsx

```diff
@@ -37,6 +38,7 @@ export interface MetabaseProviderProps {
   eventHandlers?: SdkEventHandlersConfig;
   theme?: MetabaseTheme;
   className?: string;
+  locale?: string;
```

Comment:
I'm wondering if `locale` could be in the SDK config object, instead of in the provider's props. I think it's fine to keep it here.